template2.html

```diff
@@ -0,0 +1,180 @@
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+                 </p>
+            </div>
+            <footer>
+               <div id="skills_1" class="skills">
+                 <p>html</p>
+                 <p>css</p>
+                 <p>javaScript</p>
+               </div>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+              </footer>
+  
+            
+          </section>
+          <section class="p">
+            <div>
+                  <img  class="project_image" src="statics/images/Snapshoot_Portfolio2.png" alt="snapshoot image project">
+                  <h1>Tonic</h1>
+                  <div id="general_inf_2">
+                      <p class="title">CANOPY</p>
+                      <img class="counter" src="statics/images/Counter.png" alt="counter"/>
+                      <p class="job-title">Back End Dev</p>
+                      <img class="counter" src="statics/images/Counter.png" alt="counter">
+                      <p class="year">2015</p>
+                  </div>
+              
+                 <p class="project_description">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+                 </p>
+            </div>
+            <footer>
+               <div id="skills_2" class="skills">
+                 <p>html</p>
+                 <p>css</p>
+                 <p>javaScript</p>
+               </div>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+              </footer>
+  
+            
+          </section>
+          <section class="p">
+            <div>
+                  <img  class="project_image" src="statics/images/Snapshoot_Portfolio3.png" alt="snapshoot image project">
+                  <h1>Multi-Post
+                    Stories</h1>
+                  <div id="general_inf_3">
+                      <p class="title">CANOPY</p>
+                      <img class="counter" src="statics/images/Counter.png" alt="counter"/>
+                      <p class="job-title">Back End Dev</p>
+                      <img class="counter" src="statics/images/Counter.png" alt="counter">
+                      <p class="year">2015</p>
+                  </div>
+              
+                 <p class="project_description">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+                 </p>
+            </div>
+            <footer>
+               <div id="skills_3" class="skills">
+                 <p>html</p>
+                 <p>css</p>
+                 <p>javaScript</p>
+               </div>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+              </footer>
+  
+            
+          </section>
+          <section id="about_me">
```

Comment:
- [ ] The about section should have its content aligned using flexbox instead of grid. Please consider changing it.